### PR TITLE
internal api for cristin service

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -333,3 +355,20 @@ Resources:
       RestApiId: !Ref NvaCristinProxyApi
       Stage: !Ref NvaCristinProxyApi.Stage
 
+  NvaCristinProxyInternalApiBasePathMapping:
```
Comment (torarnet) [inline template.yaml:358]: Will this crash with NvaCristinProxyBasePathMapping?

File: docs/cristin-proxy-internal-apidoc.yaml [hunk 2]
```diff
@@ -0,0 +1,1074 @@
...
+          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${NvaCristinFetchPersonUsingNinFunction.Arn}/invocations
+        httpMethod: POST
+        type: "AWS_PROXY"
+      tags:
+        - Person
+      summary: Query person by national identity number
+      description: Returns a single person matching nin query parameter
+      operationId: GetPersonByNIN
+      security:
+        - CognitoUserPool: ['https://api.nva.unit.no/scopes/backend']
```
Comment (torarnet) [inline docs/cristin-proxy-internal-apidoc.yaml:41]: So this is where the magic happens? Have you tested it locally? Should you write karate tests for it? Do you know if it works?

**Review by torarnet — COMMENTED** [review]
See comments

File: template.yaml (hunk 1 above)
Comment (axthosarouris) [reply template.yaml]: fixed

**Review by svenngjeto — APPROVED** [review]

**Review by torarnet — APPROVED** [review]

**Pull request merged**